app/views/conference_registrations/_form.html.haml

```diff
@@ -22,6 +22,7 @@
             = render partial: 'devise/shared/sign_up_form_embedded'
 
             = render partial: 'registration_info', locals: { f: f }
+            = f.input :conference_id, as: :hidden, value: @conference.id
```

Comment:
It's not, but AFAICT it's missing from the form, and just plain broken without it.

Reply:
Would you mind putting it in a different commit? I think it will be helpful in the future to separate those two (someone may want for example to pick the commit with the bug fix without removing the travel information or we may debug this code in a year and wonder how is this related).

Reply:
I've removed that line from the commit; it's not _required_ in upstream `osem` at this time, but I do recall now why I need it, and why it's a good practice.

Upstream, the conference is being identified using routing parameters. AT LFNW, we're not.

In general, using routing parameters in form data creates a non-obvious dependency of any form submission on the routing table; e.g. changing a route can cause a form submission to fail, unexpectedly. I find it to be more obvious, and a clear separation of responsibilities, to simply include all necessary attributes in the form data; routing is therefore limited to the responsibility of (surprise) routing, and the form data is complete for form submission, and neither affects the other.

Reply:
> It seems something went wrong while rebasing. Tests are failing and the commit has a strange message

The tests require that form element, for context :doh:

Reply:
I restored the form element, which _should_ allow the strong params to work again.